### PR TITLE
Doc:Fixes Python Swift client commands

### PR DESCRIPTION
**Pull request by jronak**
issue: http://tracker.ceph.com/issues/17746
signed-off-by: Ronak Jain <ronakjain@outlook.in>

### CODE REVIEW

File: doc/dev/quick_guide.rst [hunk 1]
```diff
@@ -117,9 +117,9 @@ You can now use the swift python client to communicate with the RadosGW.
 
 .. code::
 
-    $ swift -A http://localhost:8000/auth -U tester:testing -K asdf list
-    $ swift -A http://localhost:8000/auth -U tester:testing -K asdf upload mycontainer ceph
-    $ swift -A http://localhost:8000/auth -U tester:testing -K asdf list
+    $ swift -A http://localhost:8000/auth -U test:tester -K testing list
+    $ swift -A http://localhost:8000/auth -U test:tester -K testing upload my container ceph
```
Comment (cbodley) [inline doc/dev/quick_guide.rst]: still needs to be `mycontainer`, right?

Comment (jronak) [inline doc/dev/quick_guide.rst]: My bad, I have updated the pr.

**Comment by cbodley** [comment]
thanks. could you please squash the two commits together, and make sure the commit message to say `Fixes: http://tracker.ceph.com/issues/17746`?

**Comment by jronak** [comment]
Done, please check now

**Pull request merged**